### PR TITLE
feat: generate valid passwords when using additional validators

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description

Migration of https://github.com/edunext/edx-platform/commit/33958fb0e7c78bb620dee53ad1e509d917a1c093

### CODE REVIEW

**Review by johanv26 — COMMENTED** [review]
At the difference with the original commit https://github.com/eduNEXT/edx-platform/commit/33958fb0e7c78bb620dee53ad1e509d917a1c093  why do we have to load generate password from another side?

Does `from edx_django_utils.user import generate_password`
not have the generate_password method would we change it?

**Comment by johanv26** [comment]
Ok, I check and edx_django_utils is like a package. 
https://github.com/openedx/edx-django-utils/blob/c80c15029dca78ee6cb01bccac17501e7c7a7409/edx_django_utils/user/__init__.py
Thinking in that this would be the fast   solution 🤔to modify the generate_password method.

**Comment by johanv26** [comment]
No, that this PR is changing the import from the package. That is the reason you have to add the imports.
If this would go to upstream the change will be proposed to the package edx-django-utils.

**Review by johanv26 — APPROVED** [review]

**Comment by andrey-canon** [comment]
> No, that this PR is changing the import from the package. That is the reason you have to add the imports. If this would go to upstream the change will be proposed to the package edx-django-utils.

yes, there are two options this pr and clone the edx-django-utils and modify that library but I decided that this solution is faster and easier to maintain, we should think a solution that covers our requirements and works for the community but that will be future work.

File: openedx/core/djangoapps/user_authn/views/register.py [hunk 1]
```diff
@@ -55,7 +55,7 @@
 from openedx.core.djangoapps.user_api.preferences import api as preferences_api
 from openedx.core.djangoapps.user_authn.cookies import set_logged_in_cookies
 from openedx.core.djangoapps.user_authn.utils import (
-    generate_username_suggestions, is_registration_api_v1
+    generate_password, generate_username_suggestions, is_registration_api_v1
```
[the remark this answers is not in the document]
Comment (andrey-canon) [reply openedx/core/djangoapps/user_authn/views/register.py:58]: I'm trying to make the minimal changes, personally I don't like that I prefer option  3(Vertical Hanging Indent ) https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html  however those changes are not related with this pr

**Pull request merged**